src/components/common/CommonInput.js

```diff
@@ -0,0 +1,18 @@
+import { string } from "prop-types";
+
+import "./common-input.scss";
+
+const CommonInput = ({ labelText, inputType, placeHolderText }) => (
```

Comment:
You don't need to name these props with 'text' postfix when they are props developers already know their meaning. Label and placeholders are essentially only texts